x-pack/plugins/apm/public/components/shared/dependencies_table/index.tsx

```diff
@@ -131,9 +133,29 @@ export function DependenciesTable(props: Props) {
     },
     {
       field: 'errorRateValue',
-      name: i18n.translate('xpack.apm.dependenciesTable.columnErrorRate', {
-        defaultMessage: 'Failed transaction rate',
-      }),
+      name: (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.dependenciesTable.columnErrorRateTip',
+            {
+              defaultMessage:
+                "The percentage of failed transactions for the selected service. HTTP server transactions with a 4xx status code (client error) aren't considered failures because the caller, not the server, caused the failure.",
+            }
+          )}
+        >
+          <>
+            {i18n.translate('xpack.apm.dependenciesTable.columnErrorRate', {
+              defaultMessage: 'Failed transaction rate',
+            })}{' '}
```

Comment:
same here with the extra space character:

```suggestion
            })}
```